content/support.rst

```diff
@@ -0,0 +1,193 @@
+Support
+#######
+
+:menu_order: 009
+
+.. contents::
+
+
+Introduction
+============
+
+This page describes the guidelines for the support activities together with
+best practices that were extracted based on our past experience with support
+related activities.
+
+For access to any of the Support platforms, please contact the Support team.
+
+For the full details on the Support offering, please go to the website's
+`Support options <https://www.sftpplus.com/support/options.html>`_ page.
+
+
+Helpdesk
+========
+
+The main point of contact for Support and liaising with customers
+is via the Google Group email.
+Each message to the Support email automatically creates a helpdesk ticket.
```

Comment:
I think it's early days to write this sentence.
Because we are looking at archiving emails, as well as new software. 
I am taking this out, plus any references about making tickets.